code/modules/projectiles/guns/energy/stun.dm

```diff
@@ -10,7 +10,9 @@
 	name = "hybrid taser"
 	desc = "A dual-mode taser designed to fire both short-range high-power electrodes and long-range disabler beams."
 	icon_state = "advtaser"
-	ammo_type = list(/obj/item/ammo_casing/energy/electrode, /obj/item/ammo_casing/energy/disabler)
+	weapon_weight = WEAPON_MEDIUM //BUBBER ADDITION
+	w_class = WEIGHT_CLASS_NORMAL //BUBBER ADDITION
```

Comment:
if you do this kill /obj/item/gun/energy/e_gun/advtaser/normal, and modularize these changes